share/container/docker.go

```diff
@@ -546,12 +524,30 @@ func (d *dockerDriver) MonitorEvent(cb EventCallback, cpath bool) error {
 		handler = d.eventHandler
 	}
 
-	errCh := make(chan error)
+	ctx, cancelCtx := context.WithCancel(context.Background())
+	d.eventCancel = cancelCtx
+
+	var bRunning, bTimeoutError bool
 	for {
-		go d.client.StartMonitorEvents(handler, errCh)
+		msgs, errCh := d.client.Events(ctx, dockerTypes.EventsOptions{})
```

Comment:
The --since and --until parameters can be Unix timestamps, date formatted timestamps, or Go duration strings (e.g. 10m, 1h30m) computed relative to the client machine’s time. If you do not provide the --since option, the command returns only new and/or live events.

docker events --since 10m
==> This command will show events that occurred within the last 10 minutes.

So far, I did not see the event duplicates without this "Since" option.